caravel/assets/visualizations/histogram.js

```diff
@@ -0,0 +1,132 @@
+// JS
+var d3 = require('d3')
+var px = window.px || require('../javascripts/modules/caravel.js')
+
+// CSS
+require('./histogram.css')
+
+function histogram(slice) {
+    
+    var div = d3.select(slice.selector)
+    
+    var _draw = function(data, noOfBins) {
+        
+        var margin = { top: 50, right: 10, bottom: 20, left: 50 },
```

Comment:
we are using the airbnb open source javascript style guide in this code base now, could you update to follow these guidelines? 

https://github.com/airbnb/javascript

a few that jump out: 
- single variable per line 
- use const or let
- use single quotes in js
- use trailing commas

we have an automatic linter happening on all js EXCEPT visualizations, but it would be great for all new visualizations to use this code style. thx!